src/utils/isEmptyArray.js

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+/**
+ * Returns whether the value is a empty array.
+ * @param {*} value
+ * @returns {boolean}
+ */
+export default value => Array.isArray(value) && value.length === 0;
```

Comment:
No need for this util because unprocessedDecisions will always be an array.

Reply:
thank you, make sense